pkg/nb/types.go

```diff
@@ -571,6 +571,24 @@ type DeleteNamespaceResourceParams struct {
 	Name string `json:"name"`
 }
 
+// UpdateAccountParams is the params of account_api.update_account_s3_access()
+type UpdateAccountParams struct {
```

Comment:
I think that you can ignore optional params that are not relevant to the operator and were mainly used by the UI. Specifically `MustChangePassword` and `Preferences`